examples/reorderable_list_widget_example.py

```diff
@@ -1,36 +1,31 @@
 from eqt.ui.ReOrderableListWidget import ReOrderableListWidget
 from PySide2 import QtWidgets
 import sys
-try:
-    import qdarkstyle
-    from qdarkstyle.dark.palette import DarkPalette
-    HAS_QDARKSTYLE = True
-except:
-    HAS_QDARKSTYLE = False
```

Comment:
@lauramurgatroyd any reason you had this as an optional dependency before?

Reply:
I am not certain, but I think @paskino previously advised that we should minimise the number of required dependencies

Reply:
hmm. `qdarkstyle` is on `conda-forge` (at least as of now) so perhaps no longer an issue @paskino?